test/built-ins/Proxy/deleteProperty/targetdesc-is-configurable-target-is-not-extensible.js

```diff
@@ -0,0 +1,32 @@
+// Copyright (C) 2019 Aleksey Shvayka. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+/*---
+es6id: 9.5.10
+description: >
+    Throw a TypeError exception if trap result is true, targetDesc is configurable,
+    and target is not extensible.
+info: |
+    [[Delete]] (P)
+
+    ...
+    13. Let extensibleTarget be ? IsExtensible(target).
+    14. If extensibleTarget is false, throw a TypeError exception.
+    ...
+features: [Proxy]
+---*/
+
+var target = {
+  prop: 1,
```

Comment:
also add an assertion for a property that does not exist. If the trap returns true, and the target desc is undefined, it will return true. This is still valid if target is not extensible. You can catch that using Reflect as well.

Reply:
5d510ad